extensions/amp-subscriptions/0.1/test/validator-amp-subscriptions-errors.html

```diff
@@ -27,7 +27,7 @@
   <script async src="https://cdn.ampproject.org/v0.js"></script>
   <script async custom-element="amp-subscriptions" src="https://cdn.ampproject.org/v0/amp-subscriptions-0.1.js"></script>
   <script async custom-element="amp-subscriptions-google" src="https://cdn.ampproject.org/v0/amp-subscriptions-google-0.1.js"></script>
-  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
```

Comment:
since this is the test file for errors, perhaps leaving the version as 0.1 to capture that the deprecation warning is working correctly? or can add a separate file for it.

Reply:
Good point. I actually added `validator-amp-mustache-version.html` for both 0.1 and 0.2 to test this.